filesender/main.py

```diff
@@ -192,7 +198,7 @@ async def server_info(
     """Prints out information about the FileSender server you are interfacing with"""
     client = FileSenderClient(base_url=context.obj["base_url"])
     result = await client.get_server_info()
-    print(result)
+    logger.log(LogLevel.INFO.value, result)

```

Comment:
Unimportant question:

Just checking, inconsequential but with something like the `server-info` command, did you mean that to be using the logging module when outputting the result to `stdout`?

```
(venv) [u.ja@slurm-login01 FileSenderCli]$ filesender --base-url https://filesender.aarnet.edu.au/rest.php server-info
[13:43:29] INFO     HTTP Request: GET                            _client.py:1786
                    https://filesender.aarnet.edu.au/rest.php/in
                    fo "HTTP/1.1 200 OK"
           INFO     {'url': 'https://filesender.aarnet.edu.au/',     main.py:201
                    'name': 'AARNet FileSender', 'logon_url':
                    'https://filesender.aarnet.edu.au/simplesaml/mod
                    ule.php/saml/sp/login/default-sp?ReturnTo=__targ
                    et__AuthId=default-sp&ReturnTo=__target__',
                    'upload_chunk_size': 12582912}
```

Rather than previously when it was `print()` you just get the JSON directly:

```
❯ venv/bin/filesender --base-url https://filesender.aarnet.edu.au/rest.php server-info
{
    'url': 'https://filesender.aarnet.edu.au/',
    'name': 'AARNet FileSender',
    'logon_url': 'https://filesender.aarnet.edu.au/simplesaml/module.php/saml/sp/login/default-sp?ReturnTo=__target__AuthId=default-sp&ReturnTo=__target__',
    'upload_chunk_size': 12582912
}
```

Reply:
This is a good point. I'm not really sure how to structure this. If I use the logger then it can't go to stdout or be easily consumed by other apps, but if I use print statements, then I don't get the configurability of using a logger: log levels and log handlers

Reply:
I think conceptually it's all "logging", and that if the user wants to consume the metadata, they should use the Python API.